src/browsercheck.js

```diff
@@ -83,5 +87,9 @@ if ($BROWSERS.length) {
   var supported = isSupported($BROWSERS, navigator.userAgent);
   if (!supported) {
     document.getElementById('browser-warning').style.display = 'block';
+    if (navigator.userAgent.includes('Steam')) {
+      document.getElementById('browser-warning').innerText +=
+        " If DIM doesn't work in the Steam browser but does work in other browsers, contact Valve.";
```

Comment:
```suggestion
        " It looks like this page is loaded in Steam's browser. Due to its limited features and resources, it may unexpectedly or intermittently fail to run DIM. We cannot provide support for it.";
```

Reply:
I had this as a `+=` to the existing message, do you think it should be a replacement?

Reply:
maybe replacing and therefore keeping it shorter, would increase the chances of someone reading it.